docs/source/glossary.md

```diff
@@ -3,18 +3,22 @@
 A glossary of common terms used throughout the PyMC3 documentation and examples.
 
 :::::{glossary}
-[Term with external reference](https://www.youtube.com/watch?v=6dc7JgR8eI0)
-  Terms are defined within this glossary directive. The term id is defined as the non
-  indented line, and can be text alone (like {term}`second term`) or also include a link
-  to an external reference.
+[Equidispersion](http://www.ce.memphis.edu/7012/L20_CountDataModels_v2.pdf)
+  If in a Poisson distribution if the variance equals the mean of the distribution, it is reffered to as equidispersion.
 
-Second term
-  Definitions can have as many lines as desired, and should be written in markdown. Definitions
-  can contain any markdown formatting for MyST to parse, this includes basic markdown like **bold**
-  as well as MyST directives and roles like {fa}`fort awesome,style=fab`
-Functional Programming
-  Functional programming is a programming style that prefers the use of basic functions with explicit and distinct inputs and outputs.
-  This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
-Dispatching
-  Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
-:::::
+[Generalized Poisson PMF](https://www.jstor.org/stable/1267389)
+  A generalization of the {term}`Poisson distribution`, with two parameters X1, and X2, is obtained as a limiting form of the {term}`generalized negative binomial distribution`. The variance of the distribution is greater than, equal to or smaller than the mean according as X2 is positive, zero or negative. For formula and more detail, visit the link in the title.
+
+[Bayes' theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem)
+  Describes the probability of an event, based on prior knowledge of conditions that might be related to the event. For example, if the risk of developing health problems is known to increase with age, Bayes' theorem allows the risk to an individual of a known age to be assessed more accurately (by conditioning it on their age) than simply assuming that the individual is typical of the population as a whole.
+  Formula:
+  {term}`P(A|B) = (P(B|A) P(A))/P(B)`
```

Comment:
This will not work, `term` is only to link to terms defined in the glossary. Math can be inserted either as inline equation or block equation with dollar delimiters like one would do on latex/jupyter notebook (reference: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#dollar-delimited-math, we already have everything configured for the repo, so pay attention only to the examples)